### DISCUSSION TITLE
ebusd on Home Assistant: No connection to device

### DISCUSSION THREAD

**Discussion by Pantalooney** [john30/ebusd#932] [Q&A]
Hi, I have:

- an ebus adapter v5, flashed with the latest hardware, reachable via LAN (WiFi).
- Home Assistant Supervisor in its newest version 
- installed the ebusd HA add-on from Lukas Grebe: https://github.com/LukasGrebe/ha-addons
- this ebusd integration 
- added the following to my configuration.yaml:
ebusd:
  host: 127.0.0.1
  circuit: "700"

In the configuration GUI of the Add-on, I have tried all three modes and each time added  xxx:192.168.178.40:9999 as network adapter address, with xxx being the selected mode. 

None of these helped, I keep on getting the following error:

```
Logger: homeassistant.setup
Source: components/ebusd/__init__.py:69
First occurred: 17:50:03 (1 occurrences)
Last logged: 17:50:03

Error during setup of component ebusd
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/ebusdpy/ebusdpy.py", line 14, in init
    sock.connect(address)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 286, in _async_setup_component
    result = await task
             ^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/ebusd/__init__.py", line 69, in setup
    ebusdpy.init(server_address)
  File "/usr/local/lib/python3.11/site-packages/ebusdpy/ebusdpy.py", line 18, in init
    raise EBusError(socket.error)
ebusdpy.ebusdpy.EBusError: <class 'OSError'>
```

Any idea what might be wrong here?

**Comment by Koky05** [comment]
As network adapter you should use same string as is shown in ebus adapter, for example `network_device: ens:192.168.178.40:9999`

It is mentioned in Addon [documentation](https://github.com/LukasGrebe/ha-addons/blob/main/ebusd/DOCS.md#how-to-run-ebusd)

**Reply by Pantalooney** [reply]
Hi,
This was what I had done. 

The reported issue is resolved now - unfortunately I do not remember how, it's more than 6 months ago. 

But thanks for trying to help!
